app/models/planning_application.rb

```diff
@@ -175,6 +175,17 @@ class WithdrawOrCancelError < RuntimeError; end
     numericality: {greater_than_or_equal_to: 0, less_than_or_equal_to: 1_000_000},
     allow_nil: true
 
+  validate do
+    errors.add(:application_type_id, :blank) unless application_type
+  end
+
+  with_options format: {with: URI::MailTo::EMAIL_REGEXP} do
```

Comment:
For the future but we probably should add a email validator that matches Notify - I think this will allow some that Notify rejects.